public/locales/en-US/translations.json

```diff
@@ -214,6 +214,7 @@
   "detailed": "Detailed",
   "details": "Details",
   "history": "History",
+  "voting": "Voting",
```

Comment:
Can you add `null` values to the other `translations.json` files? So it'll be easier for translators to know what they're missing.